packages/profile-sync-controller/src/shared/encryption/encryption.ts

```diff
@@ -252,9 +264,23 @@ class EncryptorDecryptor {
     }
 
     const newSalt = salt ?? SHARED_SALT;
+    const keyRef = `${hashedPassword}${bytesToHex(newSalt)}`;
+    if (keyStore) {
+      try {
+        const storedKey = await keyStore.loadKey(keyRef);
```

Comment:
Oooh very cool, so this keyStore will be used to store the encrypted keys to be used by other potential platforms?

As we persist this on the extension, would we need a way for this to be accessible outside of the extension? Reason I'm asking is could we place this keystore into the snap storage? That way sites can access this from the snap rather than the extension?